code/modules/mining/lavaland/loot/hierophant_loot.dm

```diff
@@ -49,42 +50,50 @@
 
 /obj/item/hierophant_club/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
 	..()
-	var/turf/T = get_turf(target)
-	if(!T || timer > world.time)
-		return
-	calculate_anger_mod(user)
-	timer = world.time + CLICK_CD_MELEE //by default, melee attacks only cause melee blasts, and have an accordingly short cooldown
-	if(proximity_flag)
-		INVOKE_ASYNC(src, .proc/aoe_burst, T, user)
-		if(is_station_level(T.z))
-			add_attack_logs(user, target, "Fired 3x3 blast at [src]")
-		else
-			add_attack_logs(user, target, "Fired 3x3 blast at [src]", ATKLOG_ALL)
-	else
-		if(ismineralturf(target) && get_dist(user, target) < 6) //target is minerals, we can hit it(even if we can't see it)
-			INVOKE_ASYNC(src, .proc/cardinal_blasts, T, user)
-			timer = world.time + cooldown_time
-		else if(target in view(5, get_turf(user))) //if the target is in view, hit it
+	if(world.time > timer)
```

Comment:
An early return is preferred in this codebase.
Early returns are like this:
```
if(!can_continue)
  return
if(!other_check)
  return
do_stuff
```

This way the code stays on rougly the same indentation which improves readability

Reply:
Done